projects/igniteui-angular/src/lib/grids/grid/grid.component.ts

```diff
@@ -163,6 +163,49 @@ export class IgxGridComponent extends IgxGridBaseComponent implements IGridDataB
         }
     }
 
+    /**
+     * Returns the state of the grid virtualization, including the start index and how many records are rendered.
+     * ```typescript
+     * const gridVirtState = this.grid1.virtualizationState;
+     * ```
+	 * @memberof IgxGridBaseComponent
```

Comment:
@skrustev It's not longer member of `IgxGridBaseComponent`. Please fix it for all props.